models/addresses/ethereum/addresses_ethereum_bridges.sql

```diff
@@ -5,9 +5,7 @@
                                     \'["hildobby"]\') }}')}}
 
 SELECT address, bridge_name, description
-FROM (VALUES
-    -- Source: https://etherscan.io/accounts/label/bridge
```

Comment:
We can put this comment above the select maybe?

Reply:
👍 done